### PR TITLE
display alert icon when sync failed for 24 hours

### PR DESCRIPTION
**Pull request by ericcf**
[#92632922]

### CODE REVIEW

File: test/unit/controllers/ActionsControllerSpec.js [hunk 1]
```diff
@@ -16,8 +16,14 @@ describe('ActionsController', function() {
...
       };
     };
+    var mockSynchronizer = function() {
+      this.isAlertable = function() {
+        return false;
+      };
+    };
 
     $provide.service('dialogResponses', mockDialogResponses);
+    $provide.service('synchronizer', mockSynchronizer);
```
Comment (michaelwehrley) [inline test/unit/controllers/ActionsControllerSpec.js]: Did you intend to use this for testing?

File: src/index.html [hunk 2]
```diff
@@ -19,6 +19,7 @@
         MedLink
         <span id="post-header"></span>
         <span id="header-actions" style="display: inline-block; float: right;">
+          <i ng-if="actions.isSyncAlertable()" class="icon-exclamation-sign" style="color: pink;">&nbsp;</i>
```
Comment (michaelwehrley) [inline src/index.html]: :wolf: sometimes you just have to use inline styles :smile:

File: test/unit/controllers/ActionsControllerSpec.js (hunk 1 above)
Comment (ericcf) [inline test/unit/controllers/ActionsControllerSpec.js]: No, I just wanted to stub out dependencies to keep it isolated.

**Comment by michaelwehrley** [comment]
:+1:

**Pull request merged**